Topics/Tech_Stacks/Framer_Motion.md

```diff
@@ -0,0 +1,122 @@
+# Intro to Framer Motion, an Animation Library for React
+
+## Table of contents
+### [Introduction & Prerequisites](#introduction-&-prerequisites)
+### [Installation](#installation)
+### [Framer Motion Basics](#framer-motion-basics)
+- #### [The Motion Component](#motion)
+- #### [Animation Props](#animation-props)
+- #### [Transitions](#transitions)
+- #### [Keyframes](#keyframes)
+- #### [Variants](#variants)
+### [Conclusion](#conclusion)
+### [Resources](#resources)
+
+
+## Introduction & Prerequisites
+Framer Motion is a production-ready animation library designed for React. Framer Motion simplifies the process of adding animations by using unique components and props which act as an interface for adding animated properties to elements. 
+
+This page will serve as quick, easy, and informative guide to getting started with the Framer Motion animation library. It will cover the first time installation and set-up of Framer Motion, foundational knowledge of animation components & props, an overview of a few key modifiable values, and a couple of quick tricks. A list of resources will also be included at the end. 
```

Comment:
You forgot the "a" on line 19. The sentence should be "This page will serve as a quick, easy, and informative guide..."

Reply:
[6b79590](https://github.com/learning-software-engineering/learning-software-engineering.github.io/pull/515/commits/6b79590bb3a70e6ab569d02509f473cfc1fc4162)